Godeps/Godeps.json

```diff
@@ -1,7 +1,7 @@
 {
 	"ImportPath": "k8s.io/kubernetes",
 	"GoVersion": "go1.9",
-	"GodepVersion": "v79",
+	"GodepVersion": "v80",
```

Comment:
Please don't change the godep version, or at least, do it in a separate PR and update all documentation, too.

Reply:
agree